test/test_mps.py

```diff
@@ -9670,7 +9670,9 @@ class TestConsistency(TestCase):
         "true_divide"
     }
 
```

Comment:
Was there an issue with the previous path ?

Reply:
Yes, invoking `test_mps.py` directly from any other folder than pytorch's main folder would not work (the path was relative to the main folder: `./test/cuda_results.yaml`).
E.g invoking it directly from `pytorch/test/` using `python3 test_mps.py` wouldn't work (`FileNotFoundError: [Errno 2] No such file or directory: './test/cuda_results.yaml'`).

Reply:
Looks good, thanks Denis, this will resolve recent failures from OATS.